packages/login/src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react'
+import {FormattedMessage, intlShape} from 'react-intl'
+import {Pages} from '../../types/Pages'
+import * as Tocco from 'tocco-ui'
+
+import '../Login/styles.scss'
+
+export class LoginForm extends Component {
+
+  handleSubmit(e) {
+    e.preventDefault()
+    this.props.login(this.props.username, this.props.password)
+  }
+
+  handleUsernameChange(e) {
+    this.props.setUsername(e.target.value)
+  }
+
+  handlePasswordChange(e) {
+    this.props.setPassword(e.target.value)
+  }
+
+  render() {
+    return (
+      <div className="login-form">
+        {
+          this.props.showTitle
+          && <div>
+            <h1><FormattedMessage id="client.login.form.title"/></h1>
+            <p><FormattedMessage id="client.login.form.introduction"/></p>
+          </div>
+        }
+        <form>
```

Comment:
I think we should register the `handleSubmit` event handler on the form (not as click handler on the button). And the submit button should be of type `submit` (without click handler).